### ISSUE TITLE
Does non-planar region mask quality deteriorate after finetuning for depth estimation?

### ISSUE THREAD

**Issue by MichaelRamamonjisoa** [svip-lab/PlanarReconstruction#42]
Hi,

Thanks a lot for your great work! I was curious about your experiments on depth estimation.
One thing I noticed on PlaneNet is that when they finetune on NYUv2 with the **depth loss** only, they get less and less regions detected as planar (only the layout).

I was wondering if you observe a similar phenomenon in your case, and if it explains the depth performance improvement.
Could you maybe share some snapshots of [non-planar mask, instance mask, depth maps], before and after fine-tuning with your loss in Equation 10.
![image](https://user-images.githubusercontent.com/25643393/134024946-a1b7ddd4-3377-4737-a281-8089aae47b5c.png)

Thanks in advance for your help!
